python-backend/app/api/mines/permits/permit/resources/permit.py

```diff
@@ -47,7 +47,6 @@ class PermitListResource(Resource, UserMixin):
     parser.add_argument('uploadedFiles', type=list, location='json', store_missing=False)
 
     @api.doc(params={'mine_guid': 'mine_guid to filter on'})
-    @requires_role_view_all
```

Comment:
We still want to keep this role I believe.

Reply:
fixed